src/db/models/Integrations.js

```diff
@@ -1,23 +1,315 @@
+   * @param {Boolean} doc.messengerData.isOnline - Identicates whether messenger in online or not
+   * @param {Object[]} doc.messengerData.onlineHours - OnlineHours object array
+   * @param {string} doc.messengerData.onlineHours.day - OnlineHours day
+   * @param {string} doc.messengerData.onlineHours.from - OnlineHours from
+   * @param {string} doc.messengerData.onlineHours.to  - OnlineHours to
+   * @param {string} doc.messengerData.timezone - Timezone
+   * @param {string} doc.messengerData.welcomeMessage - Message displayed on welcome
+   *    TODO: need more elaborate documentation
+   * @param {string} doc.messengerData.awayMessage - Message displayed when status becomes away
+   *    TODO: need more elaborate documentation
+   * @param {string} doc.messengerData.thankYouMessage - Thank you message
+   *    TODO: need more elaborate documentation
+   * @param {string} doc.messengerData.uiOptions.color - Color of messenger
+   * @param {string} doc.messengerData.uiOptions.wallpaper - Wallpaper image for messenger
+   * @param {string} doc.messengerData.uiOptions.logo - Logo used in the embedded messenger
+   * @param {Object} doc.twitterData - Twitter data
+   *    TODO: need more elaborate documentation
+   * @param {Object} doc.facebookData - Facebook data
+   *    TODO: need more elaborate documentation
+   * @return {Promise} returns integration document promise
+   */
+  static createIntegration(doc) {
+    return this.create(doc);
+  }
+
+  /**
+   * Create a messenger kind integration
+   * @param {Object} object - Integration object
+   * @param {string} object.name - Integration name
+   * @param {String} object.brandId - Integration brand id
+   * @return {Promise} returns integration document promise
+   */
+  static createMessengerIntegration({ name, brandId }) {
+    return this.createIntegration({
+      name,
+      brandId,
+      kind: KIND_CHOICES.MESSENGER,
+    });
+  }
+
+  /**
+   * Update messenger integration document
+   * @param {Object} object - Integration main doc object
+   * @param {string} object.name - Integration name
+   * @param {string} object.brandId - Integration brand id
+   * @return {Promise} return null promise
+   */
+  static updateMessengerIntegration(_id, { name, brandId }) {
+    return this.update({ _id }, { $set: { name, brandId } }, { runValidators: true });
+  }
+
+  /**
+   * Save messenger appearance data
+   * @param {string} _id
+   * @param {Object} object - MessengerUiOptions object TODO: need more elaborate documentation
+   * @param {string} object.color - MessengerUiOptions color TODO: need more elaborate documentation
+   * @param {string} object.wallpaper - MessengerUiOptions wallpaper
+   * @param {string} object.logo - Messenger logo TODO: need more elaborate documentation
+   * @return {Promise} returns null promise
+   */
+  static saveMessengerAppearanceData(_id, { color, wallpaper, logo }) {
+    return this.update(
```

Comment:
return updated integration
await this.update ....
return this.findOne({ _id })